### PR TITLE
Run command as is in Windows even with unbuffered mode.

### PR DESCRIPTION
**Pull request by yunikkk**
Continuation of https://github.com/gojuno/commander/issues/4 .

### CODE REVIEW

File: os/src/main/kotlin/com/gojuno/commander/os/Processes.kt [hunk 1]
```diff
@@ -50,15 +50,15 @@ fun process(
 
             val command: List<String>
```
Comment (arturdryomov) [inline os/src/main/kotlin/com/gojuno/commander/os/Processes.kt]: Seems like it is possible to make it like this:

```kotlin
val command: List<String> = if (unbufferedOutput) {
    commandAndArgs
} else {
    when (os()) {
        Linux -> listOf()
        Mac -> listOf()
        Windows -> commandAndArgs
    }
}
```

File: os/src/main/kotlin/com/gojuno/commander/os/Processes.kt [hunk 2]
```diff
@@ -50,15 +50,15 @@ fun process(
...
 
-            when (unbufferedOutput) {
-                false -> command = commandAndArgs
-                true -> command = when (os()) {
+            command = when (unbufferedOutput) {
+                false -> commandAndArgs
+                true -> when (os()) {
                 // Some programs, in particular "emulator" do not always flush output
                 // after printing so we have to force unbuffered mode to make sure
                 // that output will be available for consuming.
```
Comment (arturdryomov) [inline os/src/main/kotlin/com/gojuno/commander/os/Processes.kt]: Indentation went off a bit.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: os/src/main/kotlin/com/gojuno/commander/os/Processes.kt (hunk 1 above)
Comment (yunikkk) [reply os/src/main/kotlin/com/gojuno/commander/os/Processes.kt]: Ahh, sure

**Review by arturdryomov — APPROVED** [review]

**Comment by artem-zinnatullin** [comment]
:+1:

**Comment by artem-zinnatullin** [comment]
@yunikkk your git config is broken as usual lol, fix email!

**Comment by yunikkk** [comment]
@artem-zinnatullin damn it=\

**Pull request merged**